ddtrace/internal/writer/writer.py

```diff
@@ -281,14 +281,15 @@ def _reset_connection(self):
                 self._conn.close()
                 self._conn = None
 
-    def _put(self, data, headers, client):
-        # type: (bytes, Dict[str, str], WriterClientBase) -> Response
+    def _put(self, data, headers, client, no_trace=True):
+        # type: (bytes, Dict[str, str], WriterClientBase, Optional[bool]) -> Response
         sw = StopWatch()
         sw.start()
         with self._conn_lck:
             if self._conn is None:
                 log.debug("creating new intake connection to %s with timeout %d", self.intake_url, self._timeout)
                 self._conn = get_connection(self._intake_url(client), self._timeout)
+                setattr(self._conn, "_dd_no_trace", no_trace)
```

Comment:
Should we make this attribute a constant and/or leave a comment stating that is attribute will be used in the httplib integration? This could help with maintainability